yapapi/rest/configuration.py

```diff
@@ -52,7 +52,7 @@ def __init__(
         activity_url: Optional[str] = None,
     ):
         self.__app_key: str = app_key or env_or_fail("YAGNA_APPKEY", "API authentication token")
-        self.__url = url or DEFAULT_API_URL
+        self.__url = url or os.getenv("YAGNA_API_URL") or DEFAULT_YAGNA_API_URL
```

Comment:
You could just say `os.getenv("YAGNA_API_URL", DEFAULT_YAGNA_API_URL)`

Reply:
But since it's a three-element alternative it's perhaps better to write it your way, never mind.